profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileKudosLogicImpl.java

```diff
@@ -95,7 +101,15 @@ public boolean updateKudos(String userUuid, int score, BigDecimal percentage) {
 		return false;
 	}
 	
-	
+	/**
+	 * Helper to evict an item from a cache. 
+	 * @param cacheKey	the id for the data in the cache
+	 */
+	private void evictFromCache(String cacheKey) {
```

Comment:
This is common code and could go into a helper that all of the other classes that have the same could use it from.